binding.cc

```diff
@@ -1179,6 +1180,26 @@ namespace zmq {
     NanReturnValue(NanNew<String>(version_info));
   }
 
+#if ZMQ_VERSION_MAJOR >= 4
+   static NAN_METHOD(ZmqCurveKeypair) {
+    NanScope();
+
+    char public_key [41];
+    char secret_key [41];
+
+    int rc = zmq_curve_keypair( public_key, secret_key);
+    if (rc < 0) {
+      throw std::runtime_error(ErrorMessage());
```

Comment:
I guess It should work. If all tests pass it should do 

On May 21, 2015 10:10:55 AM EEST, Ron Korving <notifications@github.com> wrote:
>> @@ -1179,6 +1180,26 @@ namespace zmq {
>>      NanReturnValue(NanNew<String>(version_info));
>>    }
>>  
>> +#if ZMQ_VERSION_MAJOR >= 4
>> +   static NAN_METHOD(ZmqCurveKeypair) {
>> +    NanScope();
>> +
>> +    char public_key [41];
>> +    char secret_key [41];
>> +
>> +    int rc = zmq_curve_keypair( public_key, secret_key);
>> +    if (rc < 0) {
>> +      throw std::runtime_error(ErrorMessage());
>
>Perhaps @kkoopa can weigh in? He knows more about this than I do.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/JustinTulloss/zeromq.node/pull/425/files#r30777472